src/main/java/frc/robot/Path.java

```diff
@@ -2,39 +2,42 @@
 import java.io.File;
 import java.io.IOException;
 import java.util.ArrayList;
-import java.util.List;
 import java.util.Scanner; 
 import java.util.logging.*;
 
 public class Path{
-    private ArrayList rightString = new ArrayList<>();
```

Comment:
Noting for future update that starting up the path can likely be noticeably faster if it was directly instantiated as an array instead of having to use a CSV. Maybe we need a python script to parse the CSV and turn it into a java array literal.